### PR TITLE
Add support for evenly spaced timestamps

### PR DESCRIPTION
**Pull request by mjdesa**
Time stamps for series now start at a specific date and come in at even intervals. Additionally support for adding a jitter to the timestamp was added.

### CODE REVIEW

**Comment by otoolep** [comment]
I think, unless you specifically want it, `jitter` should be off by default in the example files.

Also, think about better names for the example files. `moderate_burn` is an example of what I think is a better name.

File: stress/config.go [hunk 1]
```diff
@@ -175,16 +177,29 @@ type seriesIter struct {
...
+	var tick time.Duration
+	var j int
+	var err error
+
+	tick, err = time.ParseDuration(s.Tick)
+	if err != nil {
+		panic(err)
+	}
+
+	if true {
```
Comment (otoolep) [inline stress/config.go]: `if true`?

**Comment by mjdesa** [comment]
Yeah, it should definitely be off by default.

**Comment by mjdesa** [comment]
I'll make more semantic file names also

**Comment by otoolep** [comment]
We can rename the files afterwards via `git mv`. Don't worry about it now. I would disable the jitter though.

**Comment by otoolep** [comment]
So if my tick is `1s` and I enable jitter, what will I get? A random number between 0 and 1,000,000,000 (tick in nanoseconds)?

**Comment by mjdesa** [comment]
yes

**Comment by mjdesa** [comment]
@otoolep good to merge?

**Comment by otoolep** [comment]
+1

**Pull request merged**